.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+      -
+        name: Login to GHCR
```

Comment:
same here, purge the GHCR login and docker action.

Reply:
Done.